backend/routes/application/application.go

```diff
@@ -0,0 +1,97 @@
+package routes
+
+/*
+/api/v1/application
+GET: Retrieve a list of applications with the most recently updated statuses (or another subset, via query params)
+
+/api/v1/application/(id)
+GET: Retrieve details on a particular application
+
+ToDos (in future): Write Operations (POST, PUT, DELETE)
+*/
+
+import (
+	"errors"
+	"log"
+	"net/http"
+	"time"
+
+	restful "github.com/emicklei/go-restful/v3"
+)
+
+// Creating a REST layer for application - ApplicationResource
+
+type Application struct {
+	Id, Name string
```

Comment:
You can add json tags to these fields (Id, Name), like this:
https://github.com/argoproj-labs/applicationset/blob/0edd1527bb73fa3c8e8578a1a6124e71412f5403/api/v1alpha1/applicationset_types.go#L45

This will allow `json.Marshal` to convert this struct to/from a JSON string.

(And likewise with the other resources)

Reply:
Yes, totally makes sense, corrected at relevant implementations!